tests/unit/store/settings.spec.js

```diff
@@ -394,15 +397,20 @@ describe("store/settings", () => {
     store.commit("settings/set_customer_account_ids", current_customerids);
 
     /*  assert the uuid and nickname from settings store is updated */
-    let updated_focus_customerid = store.getters["settings/customer_details"];
-    expect(updated_focus_customerid.uuid).toEqual("7N42dgm5tFLK9N8MT7fHC7");
-    expect(updated_focus_customerid.nickname).toEqual("Updated account -2");
+    const updated_focus_customerid =
+      store.state.settings.customer_account_ids[1];
+    expect(updated_focus_customerid.uuid).toStrictEqual(
+      "7N42dgm5tFLK9N8MT7fHC7"
+    );
+    expect(updated_focus_customerid.nickname).toStrictEqual(
+      "Updated account -2"
+    );
   });
-  test("Store a multiple customer detail with multiple user ids can be assigned to the customer_ids array an 'Edit Customer' ==>  PopopUpDialog (DeleteID) ==> SettingsForm (SaveChanges) ", () => {
+  test("Given the store has multiple customer details, When the user deletes one Customer details, Then validate the number of the customer decrements by one", () => {
```

Comment:
same comment as previous test.  The user is not doing anything here.  The test itself is calling splice and doing other things.  All that's being tested is committing a mutation to the store.  You could create a new mutation that takes an argument of which item to delete and then that mutation does the splicing.  But currently all you are testing in the main code is that committing an array to Vuex results in that array being stored. Please update the "When" section of the test to reflect what is being tested